buildSrc/src/main/java/com/uber/okbuck/core/dependency/BaseExternalDependency.java

```diff
@@ -84,7 +87,11 @@ public String packaging() {
   public String targetName() {
     StringBuilder targetName = new StringBuilder(versionless().name());
     if (isVersioned()) {
```

Comment:
add a new method `boolean usePaddedVersion()` which gets initialized based on the external dependency extension setting.